src/features/atomics.md

```diff
@@ -0,0 +1,215 @@
+---
+title: 'Atomics.wait, Atomics.notify, Atomics.waitAsync'
+author: '[Marja Hölttä](https://twitter.com/marjakh), a non-blocking blogger'
+avatars:
+  - marja-holtta
+date: 2020-09-14
+tags:
+  - ECMAScript
+  - ES2020
+description: 'Using Atomics to implement a mutex with synchronous and asynchronous modes.'
+tweet: ''
+---
+
+[`Atomics.wait`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/wait) and [`Atomics.notify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/notify) are low-level synchronization primitives useful for implementing mutexes and other means of synchronization. However, since `Atomics.wait` is blocking, it’s not possible to call it on the main thread (trying to do so will throw a TypeError).
```

Comment:
```suggestion
[`Atomics.wait`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/wait) and [`Atomics.notify`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Atomics/notify) are low-level synchronization primitives useful for implementing mutexes and other means of synchronization. However, since `Atomics.wait` is blocking, it’s not possible to call it on the main thread (trying to do so will throw a `TypeError`).
```